tests/endpoints/echo/server/app.go

```diff
@@ -278,14 +279,21 @@ func errorf(w http.ResponseWriter, code int, format string, a ...interface{}) {
 }
 
 func createServer() (*http.Server, error) {
-	var server *http.Server
 	addr := fmt.Sprintf("%v:%d", platform.GetLoopbackAddress(), *port)
+	server := &http.Server{
+		Addr: addr,
+	}
+
+	// Disable HTTP/2 support if needed by setting an empty handler.
+	if *disableHttp2 {
+		server.TLSNextProto = map[string]func(*http.Server, *tls.Conn, http.Handler){}
```

Comment:
this only works for https.  may have to check isHttps flag

Reply:
Done